features/package-install.feature

```diff
@@ -1,5 +1,33 @@
 Feature: Install WP-CLI packages
 
+  Scenario: Install a package with an http package index url in package composer.json
+    Given an empty directory
+    And a composer.json file:
+      """
+      {
+        "repositories": {
+          "wp-cli": {
+            "type": "composer",
+            "url": "http://wp-cli.org/package-index/"
+          }
+        }
+      }
+      """
+    When I run `WP_CLI_PACKAGES_DIR=. wp --info`
+    Then STDOUT should contain:
+      """
+      WP-CLI packages dir:	.
+      """
+    When I run `WP_CLI_PACKAGES_DIR=. wp package install runcommand/hook`
```

Comment:
Can we include an assertion that the "Updating package index repository URL" string is a part of the output?